stdlib/test/os/test_remove.mojo

```diff
@@ -0,0 +1,75 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+# RUN: %mojo -debug-level full %s
```

Comment:
**Suggestion** Change this to just `RUN: %mojo-no-debug %s` as we recently dropped the `-debug-level full` stuff in our stdlib tests in https://github.com/modularml/mojo/commit/b6a8f871174bde8211b43bfec50a5a7104e116df